src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java

```diff
@@ -94,6 +94,18 @@ public boolean isPendingDelete() {
         ;
     }
 
+    /**
+     * Has this computer been used to run builds?
+     */
+    public boolean isNew() {
+        /* We use the identify `connectTime == idleStartMilliseconds` as a proxy to
+        indicate that this computer hasn't run any job yet. This is not perfect
+        because if a node gets disconnected and then reconnected after being
+        used this identity will be true but it's a good enough approximation.
+        */
+        return getConnectTime() == getIdleStartMilliseconds();
```

Comment:
Relying on this is very fragile. I would say there has to be a way to advance a counter once a build / `node` block has run on a computer.